bazel/BUILD

```diff
@@ -35,6 +35,35 @@ genrule(
     stamp = 1,
 )
 
+config_setting(
+    name = "windows_x86_64",
+    values = {"cpu": "x64_windows"},
+)
+
+config_setting(
```

Comment:
Why do you define 3 different `config_setting` rules for Windows for the `compilation_mode` options? They are all used the same way, you could substitute them with `//bazel:windows_x86_64`.

Reply:
We need the 3 different `config_setting` rules for this `select` here: https://github.com/greenhouse-org/envoy/blob/windows-build-pr/bazel/envoy_build_system.bzl#L36-L41

Specifically, we want to never append `-ggdb3` to the `copts` on Windows, but we want to keep the selection mechanism on Linux. If we just add `//bazel:windows_x86_64` to that `select`, it will fail since multiple options will be true (e.g. `//bazel:windows_x86_64` and `//bazel:dbg_build`).

As far as we know, this is the best way to `select` on multiple conditions -- did we miss some other way to do this?

Reply:
Ah, I see. No, I think you're right. Thanks for the explanation!